proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+
+**Expectations**
+End users should evaluate Sandbox projects with care - some Sandbox projects may fail, and others may stay in the  Sandbox Stage indefinitely. Sandbox stage projects will receive minimal marketing support from the _$FOUNDATION_. Sandbox Stage projects will be reviewed on an annual basis; they may also request a status review by submitting a report to the CPC.
+
+**Acceptance Criteria**
+To be considered for Sandbox Stage, the project must meet the following requirements:
+* 2 CPC sponsors to champion the project & provide mentorship
+* A presentation to at the meeting of the CPC
+* Adherence to the _$FOUNDATION_ IP Policy
+* Upon acceptance, Sandbox projects must list their status prominently on website/readme
+
+### Incubating Stage*
+**Definition** 
+The Incubation Stage is for projects that are interested in reaching the Top-Level Stage, or want to have more access, participation, responsibilities and representation in the CPC. Incubating projects will receive mentorship from the CPC and are expected to actively develop their community of contributors, governance, and other project documentation. 
+
+**Examples**
+1. Projects that are on their way or very likely to become Top Level Projects.
+
+**Expectations**
+Projects can remain in the Incubating stage indefinitely, but they are generally expected to graduate within two years.
+
+**Acceptance Criteria**
+To be considered for Incubation Stage, the project must meet the Sandbox requirements as well as the following:
+ * Document that it is being used successfully in production by at least three independent end users which, in the CPC’s judgement, are of adequate quality and scope.
+ * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+ * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
+ * Recieve a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
+
+
+### Top Level/Graduate Stage*
+**Definition**
+
+**Examples**
+
+**Expectations**
+
+**Acceptance Criteria**
+To graduate from Sandbox or Incubating status, or for a new project to join as a Top-Level project, a project must meet the Incubating stage criteria plus:
+
+ * Have committers from at least two organizations.
+ * Adopt the _$FOUNDATION_ Code of Conduct.
+ * Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+ * Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
```

Comment:
I don't think this should be a requirement.

Reply:
It's also not necessarily trivial information to obtain.

Reply:
I agree with both of you, and I want to point out that this kind of information is also very helpful to member companies and foundation staff. We should think about whether there's a way to encourage this kind of information gathering via the foundation without making it a requirement here.

Reply:
This is very important information for $foundation and its marketing efforts around the projects and membership. I think it should be part of the responsibility of projects to coordinate with $foundation on gathering this info.

Reply:
We attempted to create a showcase of companies using our project on the website, but were discouraged from doing so if the companies weren't Fdn members, as that sort of exposure ought to be a perk of membership. As a smaller project, we need to be able to point at examples of commercial usage to help foster further commercial usage. Prerequing that to those companies agreeing to become members first stifled what we were able to do.

IF this clause stays, would be good to have clear guidance on it.

Reply:
@knolleary, I wonder if for Top Level projects it makes sense to be a perk of membership to highlight or prioritize work from member companies.  On the other hand at all other levels I think projects need to be able to draw from whatever references they can.

Reply:
I think projects, regardless of size, should be free to list companies on their websites that prominently/publicly use the project regardless of whether those companies are members of the $foundation. From a foundation bus. dev. standpoint I think that would be helpful, as the new ED would be able to say "hey, these companies benefit from X Project so here's a list I can target for membership!" I understand $foundation members will want their logos prioritized, but I think we can design a solution that respects and satisfies both needs.

Reply:
@jorydotcom makes sense to me.

Reply:
Ah gotcha. That sounds like an opportunity for the $foundation to provide another type of support/service to the projects! I'm going to capture your thought on the expectations/brainstorming threads.